lib/utils.ts

```diff
@@ -118,6 +118,15 @@ export function getErrorMessageFromResponse(response: BalenaRequestResponse) {
 		return errorText;
 	}
 
+	if (response.body != null && typeof response.body === 'object') {
```

Comment:
Isn't TS getting that `!response.body` from the start of this function, is already taking care of the `response.body != null` part?